src/swsssdk/configdb.py

```diff
@@ -117,16 +123,30 @@ def __typed_to_raw(self, typed_data):
                 raw_data[key] = value
         return raw_data
 
+    @staticmethod
+    def serialize_key(key):
+        string_types = str if sys.version_info[0] == 3 else basestring
```

Comment:
>string_types = str if sys.version_info[0] == 3 else basestring [](start = 8, length = 62)

in general, I found such shortcut conditional not quite readable, I prefer traditional if else condition.